independent-projects/bootstrap/gradle-resolver/pom.xml

```diff
@@ -32,4 +32,38 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
```

Comment:
Shouldn't you drop the dependency above now that you shade it?

Reply:
gradle-tooling-api? no, otherwise it won't build.